src/services

```diff
@@ -758,7 +758,6 @@ p({
       .setValidated(true)
       .setRuler(false)
       .setAuthorize(false)
-      .setApprovableAwareEnabled(false)
```

Comment:
.setApprovableAwareEnabled(false) currently exists on localCapabilityRequestDAO in net/nanopay/liquidity/services.jrl and localLiquiditySettingsDAO in src/services as well. Is there meant to be a second PR to remove all uses of .setApprovableAwareEnabled(false)

Reply:
Yes, nanopay side changes is at https://github.com/nanoPayinc/NANOPAY/pull/9810.